CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## dbt-bigquery 1.1.0 (Release TBD)
 
+### Features
```

Comment:
Nice! Could you also link the issue + PR numbers, and add a "Contributor" note, modeled on the ones below?

Note that this will be going into dbt-bigquery 1.2.0, since 1.1.0 had its final release. (The changelog update happened on the `1.1.latest` branch, and needs to be updated here as well)

Reply:
I've added links to issue, PR, and a contributor note.